.github/workflows/update-charts.yml

```diff
@@ -0,0 +1,126 @@
+                    core.exportVariable("UPDATECLI_CHART_NAME", "kubewarden-defaults")
+            } else {
+                    core.setFailed("Invalid repository")
+            }
+
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Download CRDS
+        uses: actions/github-script@v6
+        with:
+          script: |
+            let repository = context.payload.client_payload.repository
+            if (repository.endsWith("kubewarden-controller")) {
+              let crds_asset_id = context.payload.client_payload.crds_asset_id
+              console.log(`Fetching asset ID: ${crds_asset_id}`)
+              let repository_split = context.payload.client_payload.repository.split("/")
+              let owner = repository_split[0]
+              let repository = repository_split[1]
+              let asset = await github.rest.repos.getReleaseAsset({
+                      owner: owner, repo: repository, asset_id: crds_asset_id, headers:{
+                              accept: "application/octet-stream"},
+              })
+              let fs = require('fs');
+              fs.writeFileSync("crds.tar.gz", Buffer.from(asset.data))
+            }
+
+      - name: Update CRDS
+        id: update_crds
+        run: |
+          tar -xvf crds.tar.gz
+          find . -maxdepth 1 -name "*policyserver*" -exec  mv \{\} charts/kubewarden-crds/templates/policyserver.yaml \;
+          find . -maxdepth 1 -name "*admissionpolicies*" -exec  mv \{\} charts/kubewarden-crds/templates/admissionpolicy.yaml \;
+          find . -maxdepth 1 -name "*clusteradmissionpolicies*" -exec  mv \{\} charts/kubewarden-crds/templates/clusteradmissionpolicies.yaml \;
+
+          set +e
+          git diff --exit-code --no-patch charts/kubewarden-crds
+          echo "must_update_crds_chart=$?" >> $GITHUB_OUTPUT
+
+      - name: Install Updatecli in the runner
+        uses: updatecli/updatecli-action@v2
+
+      - name: Update container image tags
+        env:
+          UPDATECLI_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: "updatecli apply --config ./updatecli/updatecli.d/update-container-image-tag.yaml --values updatecli/values.yaml"
+
+      - name: Update chart values
+        run: make generate-values
+
+      - name: Update patch version of the helm chart
+        env:
+          UPDATECLI_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: "updatecli apply --config ./updatecli/updatecli.d/patch.yaml --values updatecli/values.yaml"
+
+      - name: Update CRDs Helm chart
+        if: steps.update_crds.outputs.must_update_crds_chart == 1
+        env:
+          UPDATECLI_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          UPDATECLI_CHART_NAME: "kubewarden-crds"
+        run: "updatecli apply --config ./updatecli/updatecli.d/patch.yaml --values updatecli/values.yaml"
+
+      - name: Commit changes
+        run: "updatecli/scripts/commit-changes.sh"
```

Comment:
Agree here; I'm happy with a PR that bumps crds and controller image etc, but those PRs will not be intelligent enough to know when there's new functionality that needs to be exposed in values.yaml, or when they are introducing a breaking change.
So I think bumps to `version` and `appVersion` should happen manually still.

Reply:
Also if the initial intend is commit without human validation, I would still open a pullrequest with the automerge flag enabled. 
Opening a pullrequest this way would have the following advanges

1. Visibility, as we would see the PR open and then merged in github.com/notifications
2. Test, It would trigger the test jobs which need to succeed before merging

Reply:
Okay, I'll open a PR then.